src/ert/scheduler/openpbs_driver.py

```diff
@@ -206,6 +229,20 @@ async def submit(
         logger.debug(f"Realization {iens} accepted by PBS, got id {job_id_}")
         self._jobs[job_id_] = (iens, QueuedJob())
         self._iens2jobid[iens] = job_id_
+        self._non_finished_job_ids.add(job_id_)
+
+    def _expand_truncated_jobids(
+        self, truncated_jobs: Dict[str, Dict[str, Dict[str, str]]]
+    ) -> Dict[str, Dict[str, Dict[str, str]]]:
+        """Job ids gotten through normal qstat are truncated at length 16"""
```

Comment:
Could this pose an issue?

Reply:
If so, I suppose it is fixable by cluster infrastructure, I suppose it would be in a situation where there is a long string prepended to the id, but I don't think that string can be set by the user.